appveyor.yml

```diff
@@ -0,0 +1,30 @@
+version: '{build}'
+
+pull_requests:
+  do_not_increment_build_number: true
+
+skip_tags: true
+
+image:
+- Visual Studio 2019
+
+configuration:
+- Release
+- Debug
+
+platform:
+- x64
+#- x86 # by design PS4Delta can't support x86 systems
+
+install:
+- cmd: |
+    git submodule update --init --recursive
+    set QT_DIR=C:\Qt\5.14.0\msvc2017_64
+    pmake.bat
+
+build:
+  project: build/PS4Delta.sln
+  parallel: true
+  verbosity: minimal
+
```

Comment:
If you want artifacts you'll need to add something like these:
```
after_build:
- 7z a Artifacts.zip .\build\bin\*

artifacts:
- path: Artifacts.zip
```

Reply:
Done.

Reply:
artifacts should be put at the bottom.

Reply:
I know, but here is no error, and for the future I'd like to do matrix with Windows and Linux parts and single `test: off` like here:
https://github.com/OpenXRay/xray-16/blob/xd_dev/appveyor.yml#L29

Another way: I can remove `test: off`, but I'll get such line:
`Discovering tests...OK`
Like this: https://ci.appveyor.com/project/q4a/ps4delta/builds/30504016/job/2adlynindhlx3ki9#L119

Reply:
Oh, ok